src/components/List.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import getCharacters from '../services/airbender-api';
+import { Link } from 'react-router-dom';
+
+export default class List extends Component {
+
+  state = {
+    characters: []
+  }
+
+  componentDidMount() {
+    const URL = `https://last-airbender-api.herokuapp.com/api/v1/characters?name=${this.props.match.params.id}`;
```

Comment:
there should also be functionality here to display this page with a list of all characters if no search term is provided